content/issues/4/sonorous-medieval/style.css

```diff
@@ -225,6 +226,50 @@ cite.book::before {
   background-size: 150px;
 }
 
+button.play {
+/*  mask-image: url("/img/icons/phosphor/play-circle-fill.svg");*/
+/*  border: 0;*/
+  background-color: transparent;
+  width: 30px;
+  height: 30px;
+  margin-left: -46px;
+  vertical-align: middle;
+  position: relative;
+  border-radius: 15px;  
+  border: 1px transparent;
+}
+button.play:focus-visible {
+  border: 1px dashed white;    
+  outline: none;
+}
+
+button.play:after {
+  mask-image: url("/img/icons/phosphor/play-circle-fill.svg");
+  mask-size: cover;
```

Comment:
chrome 116 didn't recognize these, but did offer to autocomplete the `-webkit-mask-image` property, so we might just need the vendor-prefixed version here too.

![Screenshot 2023-09-20 at 09 31 11](https://github.com/Princeton-CDH/startwords/assets/4924494/b7215dc6-7cb6-42a7-80fb-0e70ba672ddc)